src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -547,9 +548,15 @@ bool CurrencyDataLoader::TryParseStaticData(_In_ String^ rawJson, _Inout_ vector
     }
 
     // TODO - MSFT 8533667: this sort will be replaced by a WinRT call to sort localized strings
-    sort(begin(staticData), end(staticData), [](CurrencyStaticData unit1, CurrencyStaticData unit2)
+
+    auto loc = std::locale("");  //Use the user-preferred locale to sort country names
```

Comment:
Taking a look at PR #289 , I wonder if this should possibly be changed to get the User Locale from the  `LocalizationService` instead.
This would start working towards only having a single place in the code that references locale, which would mean that mocking (for tests) in the future would be easier.

Reply:
Additionally, the `LocalizationService` could cache the locale and collator object for perf as well.

Reply:
Related to this, maybe `LocalizationService` should have a `Sort` function which does a localized sort of a sequence of strings.